src/components/button/button_content.tsx

```diff
@@ -91,11 +91,13 @@ export const OuiButtonContent: FunctionComponent<
       <OuiLoadingSpinner className="ouiButtonContent__spinner" size="m" />
     );
   } else if (iconType) {
+    const { title } = { ...textProps };
     buttonIcon = (
       <OuiIcon
         className="ouiButtonContent__icon"
         type={iconType}
         size={iconSize}
+        title={title}
```

Comment:
I feel like we could just simplify this whole thing to

```suggestion
        title={textProps.title}
```

and delete line 94, but I won't block on that

Reply:
Good suggestion. But it could be the case that textProps is undefined. Thus I include an optional changing operator to what you suggest as follows:

``js
title={textProps?.title}
``